src/store.config.js

```diff
@@ -1,13 +1,18 @@
 import { createStore } from 'redux';
 import reducers from './reducers/index';
+import doctor from './../public/images/doctor.png';
+import lawyer from './../public/images/lawyer.png';
+import programmer from './../public/images/programmer.png';
+import teacher from './../public/images/teacher.png';
+import psychologist from './../public/images/psychologist.jpg';
 
 const storeConfigure = () => {
   const initialStore = {
     careers: [
       { title: 'Doctor',
         tagline: 'Promots and Improves Health',
         description: 'A doctor is a professional who practises medicine, which is concerned with promoting, maintaining, or restoring health.',
-        image: 'doctor.svg',
+        image: 'doctor.png',
```

Comment:
Because the images are now imported this is written as the variable name. Such as this:
`image: doctor`

Reply:
the same for the ones below